### PR TITLE
Aleksandr bohan horoscope for chat bot

### PR DESCRIPTION
**Pull request by AleksandrBohan**

### CODE REVIEW

**Review by shpaser2 — APPROVED** [review]
lgfm

File: src/main/java/org/dailyinfo/telegrambot/telegrambot/service/MessageService.java [hunk 1]
```diff
@@ -1,30 +1,102 @@
...
+        Virgo virgo  = new Virgo();
 
     public SendMessage onUpdateReceived(Update update) {
+        Aquarius aquarius = new Aquarius();
+
         SendMessage sendMessage = new SendMessage();
         if (update != null) {
             Message message = update.getMessage();
-            sendMessage.setChatId(message.getChatId());
+           // sendMessage.setChatId(message.getChatId());
```
Comment (Basist92) [inline src/main/java/org/dailyinfo/telegrambot/telegrambot/service/MessageService.java]: why you remove this String??

File: src/main/java/org/dailyinfo/telegrambot/telegrambot/service/MessageService.java [hunk 2]
```diff
@@ -1,30 +1,102 @@
...
+import org.dailyinfo.telegrambot.telegrambot.service.horoscope.*;
+import org.dailyinfo.telegrambot.telegrambot.service.horoscope.Horoscope.*;
 import org.springframework.stereotype.Service;
 import org.telegram.telegrambots.meta.api.methods.send.SendMessage;
 import org.telegram.telegrambots.meta.api.objects.Message;
 import org.telegram.telegrambots.meta.api.objects.Update;
 
 @Service
-public class MessageService {
+public class MessageService extends Horoscope{
```
Comment (Basist92) [inline src/main/java/org/dailyinfo/telegrambot/telegrambot/service/MessageService.java]: extends?? remove extends from this class

File: pom.xml [hunk 3]
```diff
@@ -62,6 +62,14 @@
 					<addResources>true</addResources>
 				</configuration>
 			</plugin>
+			<plugin>
+				<groupId>org.apache.maven.plugins</groupId>
+				<artifactId>maven-compiler-plugin</artifactId>
+				<configuration>
+					<source>14</source>
+					<target>14</target>
+				</configuration>
```
Comment (Basist92) [inline pom.xml]: why not 15

File: src/main/java/org/dailyinfo/telegrambot/telegrambot/service/MessageService.java [hunk 4]
```diff
@@ -1,30 +1,102 @@
...
+        Cancer cancer = new Cancer();
+        Capricorn capricorn = new Capricorn();
+        Gemini gemini = new Gemini();
+        Leo leo = new Leo();
+        Libra libra = new Libra();
+        Pisces pisces = new Pisces();
+        Saggittarius saggittarius = new Saggittarius();
+        Scorpio scorpio = new Scorpio();
+        Taurus taurus = new Taurus();
+        Virgo virgo  = new Virgo();
```
Comment (Basist92) [inline src/main/java/org/dailyinfo/telegrambot/telegrambot/service/MessageService.java]: annotation @Autowired  or create private method in this class for thsi strings

File: src/main/java/org/dailyinfo/telegrambot/telegrambot/service/MessageService.java [hunk 5]
```diff
@@ -1,30 +1,102 @@
...
+                }
+                if (messageText.equals("/virgo")) {
+                    return sendMessage.setText(virgo.forVirgo("Дева", 0) + virgo.forVirgo("Дева", 1) +
+                            virgo.forVirgo("Дева", 2) + virgo.forVirgo("Дева", 3));
+                }
+
             }
         }
-        return sendMessage.setText("sorry, i don`t understand you");
+        return sendMessage.setText("I don't know this horoscope!");
```
Comment (Basist92) [inline src/main/java/org/dailyinfo/telegrambot/telegrambot/service/MessageService.java]: this message - the answer from bo to client for unknown comand to bot. When i send to the bot command "/setings" - i take this message : "I don't know this horoscope!". this not good

File: src/main/java/org/dailyinfo/telegrambot/telegrambot/service/MessageService.java (hunk 1 above)
Comment (AleksandrBohan) [reply src/main/java/org/dailyinfo/telegrambot/telegrambot/service/MessageService.java]: I fixed it!

File: pom.xml (hunk 3 above)
Comment (AleksandrBohan) [reply pom.xml]: I fixed it!

File: src/main/java/org/dailyinfo/telegrambot/telegrambot/service/MessageService.java (hunk 2 above)
Comment (AleksandrBohan) [reply src/main/java/org/dailyinfo/telegrambot/telegrambot/service/MessageService.java]: I fixed it!

File: src/main/java/org/dailyinfo/telegrambot/telegrambot/service/MessageService.java (hunk 4 above)
Comment (AleksandrBohan) [reply src/main/java/org/dailyinfo/telegrambot/telegrambot/service/MessageService.java]: I fixed it!

File: src/main/java/org/dailyinfo/telegrambot/telegrambot/service/MessageService.java (hunk 5 above)
Comment (AleksandrBohan) [reply src/main/java/org/dailyinfo/telegrambot/telegrambot/service/MessageService.java]: I fixed it!

**Comment by AleksandrBohan** [comment]
I added test for Cancer's.

**Review by shpaser2 — APPROVED** [review]
Lgfm

**Review by DenisAbramovDm — APPROVED** [review]
Lgfm

**Pull request merged**